### ISSUE TITLE
Can't build bank for banker

### ISSUE THREAD

**Issue by DodgerWoman26** [teagan75/Metamorphoses#27]
Tried placing the Bank with my builders wand- it places fine and builder get the bottom stone down but when they get to wooden slabs the thing gets wonky lol. There's no graphic for the slabs but it says 72 wooden slabs. Placing the slabs in their inventory does nothing- they just keep walking back and requesting them again. Pretty much makes builder useless. Couldn't find how to cancel build either. Tried the usual- firing and hiring again- and even a different builder, same problem. I tried 2x now- both failed and each time I had to restore a backup of my game to an earlier version before I placed the bank. (Sorry I keep finding issues for you Teagan) ;)

**Comment by teagan75** [comment]
This is why I released the pack so others could help make it better. It could be a corrupted schematic. I will test it

**Comment by Ladysarajane** [comment]
I came across the same thing...I have some pics . I think when it was scanned in...too much was scanned below the structure also. 
![2019-07-25_11 47 33](https://user-images.githubusercontent.com/34258851/61891058-d1f15800-aed6-11e9-9b01-54538d9af8a1.png)

The slabs wanted for builder. Is it birch he needs? 
![2019-07-25_11 51 19](https://user-images.githubusercontent.com/34258851/61891115-f1888080-aed6-11e9-8ae3-1d5e139d03a7.png)
This is what the preview looks like. Is it aligned correctly? The builder excavated all the dirt blocks below and now requesting over 2600 wasteblocks...to me it looks like 5-6 layers of wasteland blocks he wants under it. 
![2019-07-25_11 53 34](https://user-images.githubusercontent.com/34258851/61891283-52b05400-aed7-11e9-95ec-e600394bc79f.png)

The construction tape after placing the preview...where the bottom of the schematic is being built...the wasteland blocks.  Hope this helps in tracking the problem with it.

**Comment by Ladysarajane** [comment]
Any updates on this? Now that I see my builder attempting to build this, Is there a basement to it?

**Comment by teagan75** [comment]
No updates yet. Little JJ has not given me much time this week to work on
anything. Yes, it has a basement. I will eliminate that when I redo the
schematic since it doesn't work correctly

**Comment by DodgerWoman26** [comment]
Awww... I have a 17 yrs, 13 yrs, and a 4 year old myself- totally get it!!

**Comment by Ladysarajane** [comment]
Aw....I'm sorry...I didn't mean up updates for the pack, I just meant updates on what was wrong with the bank. I was going to try to build it myself, but I have no idea where you got the schematic or ideal to build it and I couldn't see the basement inside to try to be able to duplicate it.  Yeah...in my household there is a quite a bit of activity too. A 14yo, 6yo and a 3yo. I am disabled (I have autoimmune diseases and severe lower lumbar scoliosis ( I have 5 herniated disks in that area with impingement on the  sciatic nerve which makes for interesting walking) and I am living with my brother's family. My brother is visually impaired, my sister-in-law has multiple sclerosis and it makes for an interesting family on how we mix and match our strengths and weaknesses. I know how the little ones can be. Potty training my nephew is pulling out our hair. He is a resistant twerp. And the 14yo is a typical moody teenager which is about to get worse with his first day of High School coming up. So I understand completely.  I'm sorry if I sounded pushy.  I was just confused on why the schematic put the construction tape in the ground. Never saw it place the tape in the ground before....LOL. I did have air blocks there...Just kind of odd to me, that it place them on the air blocks below solid ground.
